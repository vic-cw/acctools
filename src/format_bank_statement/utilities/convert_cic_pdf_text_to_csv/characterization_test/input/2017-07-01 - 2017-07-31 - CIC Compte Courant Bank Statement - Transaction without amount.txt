                                                                        Crédit Industriel et Commercial

                                                                                                                   CIC PARIS SAINT MARTIN

                                                                                                          224 RUE SAINT MARTIN 75010 PARIS
                                                    ¤     u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                              12345@cic.fr                                       BIC : CMCIFRPP

                                                                                                                                                                                                                 45678

                                                                                                                                                                                                                 12345

                                                                                                                           M  JACQUES  MARTIN

                                                                                                                           1  RUE     DU   CHEVAL   BLANC

                                                                                                                           75001      BERLIN

...........................................................................................................................V..O...T.R..E...C..O..N..S..E..I.L..L..E..R..:..M...S..é..b..a.s..t.i.e.n...T..O..C..A..R.D......................

                                              RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                Agence 12345

                                              31 juillet 2017

                                                    C/C CONTRAT PERSONNEL GLOBAL N° 00012345678 en euros (GD)

                                                    TITULAIRE(S) : M JACQUES MARTIN

                                              Date        Date valeur                    Opération                                                                                               Débit EUROS  Crédit EUROS

                                                                                         SOLDE CREDITEUR AU 30/06/2017                                                                                        1.234,56                                                                                       Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                              03/07/2017  03/07/2017                    PAIEMENT CB    2806 IVRY                                                                                 52,92

                                                                                        ESCROC      CARTE 00123456                                                                                                                      d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé

                                              03/07/2017  03/07/2017                    ONE LINE EXPENSE                                                                                         1,00

KV.20170801.012345.6789.1011.12131 X 0 2      10/07/2017  10/07/2017                    PAIEMENT PSC 0807 EVRY                                                                                   10,99

                                                                                        BOUTIQUE      CARTE 00123456

                                              12/07/2017  12/07/2017                    TWO LINE EXPENSE                                                                                         50,00

                                              (GE) : protégé par la Garantie de l'Etat                                                                                                                        <<Suite au verso>>

                                              (GD) : protégé par la Garantie des Dépôts

                                              Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                              www.garantiedesdepots.fr

                                                                                                                                                                                                              Page 1

                                                CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                                         M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                        Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                          RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                    Agence 12345

                                          31 juillet 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                            en  euros  (GD)

                                                TITULAIRE(S) : M JACQUES MARTIN

                                          Date        Date valeur                    Opération                                                                                               Débit  EUROS     Crédit EUROS

                                                                                    LINE TWO     CARTE 00123456

                                          10/07/2017  01/07/2017                    EXPENSE WITHOUT AMOUNT

KV.20170801.012345.6789.1011.12131 X 0 V  25/07/2017  25/07/2017                    INCOMING FUNDS                                                                                                            222,22

                                                                                    PROUT

                                          Réf : 007                                  SOLDE CREDITEUR AU 31/07/2017                                                                                            0,01

                                          QXBAN : AB01 CMCI FRPP XXX CDE2 FGHI 3JKL MNOP 4QR                               IBAN   :  FR76                                                    0123 4567 8910   1112 1314 151

                                          (GE) : protégé par la Garantie de l'Etat

                                          (GD) : protégé par la Garantie des Dépôts

                                          Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                          www.garantiedesdepots.fr

                                                                                                                                                                                                              Page 2
                                                                        Crédit Industriel et Commercial

                                                                                                            CIC SAINT MARTIN

                                                                                                  224 RUE SAINT MARTIN 75010 PARIS
                                                ¤     u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                  12345@cic.fr                                                   BIC :    CMCIFRPP

                                              RELEVE ET INFORMATIONS BANCAIRES                                                                                                                            Agence 12345

                                              31 juillet 2017

                                                SITUATION DE VOS CREDITS en euros au 31 juillet 2017

                                              Numéro                                     Libellé du compte                             Montant emprunté                                                   Solde (*)

                                              00012345678               ALLURE LIBRE                                              EUR                                                            +0,00            +0,00

                                              (*) Solde correspondant à l'amortissement normal du prêt

                                              AUTORISATIONS DE DECOUVERTS

                                              N° de compte                               Libellé                                                                                                          Montant euros

                                              00012345678               DECOUVERT C. PERS. GLOBAL TAEG 15,12% AU 28/06/2017                                                                                         50.000,00

                                              TOTAL MENSUEL DES FRAIS DEBITES SUR PRODUITS ET SERVICES BANCAIRES

                                              N° de compte                               Libellé                                                                                                          Montant euros

                                              00012345678               TOTAL DES FRAIS EN JUN 2017                                                                                                                 9,86

KV.20170801.012345.6789.1011.12131 X 0 R                                                                                                                                                                                                   d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé  Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                              (GE) : protégé par la Garantie de l'Etat

                                              (GD) : protégé par la Garantie des Dépôts

                                              Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                              www.garantiedesdepots.fr

                                                                                                                                                                                                          Page 3

                                                CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                                         M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                        Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
