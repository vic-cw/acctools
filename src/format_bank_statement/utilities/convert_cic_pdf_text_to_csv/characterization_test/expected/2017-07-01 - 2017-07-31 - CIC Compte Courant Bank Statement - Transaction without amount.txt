2017-07-03,2017-07-03,"PAIEMENT CB    2806 IVRY ESCROC      CARTE 00123456","-52,92"
2017-07-03,2017-07-03,"ONE LINE EXPENSE","-1,00"
2017-07-10,2017-07-10,"PAIEMENT PSC 0807 EVRY BOUTIQUE      CARTE 00123456","-10,99"
2017-07-12,2017-07-12,"TWO LINE EXPENSE LINE TWO     CARTE 00123456","-50,00"
2017-07-10,2017-07-01,"EXPENSE WITHOUT AMOUNT","0"
2017-07-25,2017-07-25,"INCOMING FUNDS PROUT","222,22"
